airbyte-integrations/connectors/source-clickup-api/source_clickup_api/spec.yaml

```
@@ -0,0 +1,16 @@
+documentationUrl: https://docsurl.com
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: ClickUp Api Spec
+  type: object
+  required:
+    - Authorization
+  additionalProperties: true
+  properties:
+    # 'TODO: This schema defines the configuration required for the source. This usually involves metadata such as database and/or authentication information.':
+    Authorization:
```

```suggestion
    api_token:
```
Please rename to better naming.

---

Thanks for pointing this out. I've updated this.